Atmosphere/style.css

```diff
@@ -0,0 +1,129 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  background-position: center center;
+  background-size: cover;
+  color: #fff;
+}
+
+.container {
```

Comment:
replace the class name to container-section, you  will see the signinficant change